test/hexadecimal.jl

```diff
@@ -0,0 +1,75 @@
+using Test
+using UUIDs
+using Parsers
+
+
+@testset "Hexadecimal" begin
```

Comment:
good tests: i found them easy to read through and see they were correct, as well as covering obvious edge cases